proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -142,6 +157,9 @@ service AdminService {
   // GetClusterUsage returns the summation of total message, enum and service types usage
   // for every repository in each organization within a single tenant BSR instance.
   rpc GetClusterUsage(GetClusterUsageRequest) returns (GetClusterUsageResponse);
+  // GetAllOrganizationsUsages returns the usage metrics for all repositories for every organization
+  // within a single tenant BSR instance for a given time range.
+  rpc GetAllOrganizationsUsages(GetAllOrganizationsUsagesRequest) returns (GetAllOrganizationsUsagesResponse);
```

Comment:
`All` feels like unnecessary for me, maybe something to mention in comment instead? also is this just organization instead of all owners?

Reply:
Its organization and owners. Maybe this can be called something `GetRepositoryUsages` since this just returns all the repositories and we do the groupings on the FE. wdyt?

Reply:
spoke offline, going to rename the rpc to `ListRepositoriesUsage`